### PR TITLE
Add schema based diagram

### PR DESCRIPTION
**Pull request by KayLemba**
In this PR we did the following:
- Create  file named schema_based_on_diagram.sql where we implement the database from the diagram.
- Draw ERD.
- Join tables from many-to-many relationships as shown in the diagram.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+ALTER TABLE
+    invoice_items ADD PRIMARY KEY(id);
+ALTER TABLE
+    medical_histories ADD CONSTRAINT medical_histories_patient_id_foreign FOREIGN KEY(patient_id) REFERENCES patients(id);
+ALTER TABLE
+    invoice_items ADD CONSTRAINT invoice_items_invoice_id_foreign FOREIGN KEY(invoice_id) REFERENCES invoices(id);
+ALTER TABLE
+    invoices ADD CONSTRAINT invoices_medical_history__id_foreign FOREIGN KEY(medical_history__id) REFERENCES medical_histories(id);
+ALTER TABLE
```
Comment (AshakaE) [inline schema_based_on_diagram.sql]: - [x] Kindly create indexes for all foreign keys

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @KayLemba & @Ewoskyx  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (KayLemba) [reply schema_based_on_diagram.sql]: Done. Thanks for the review @AshakaE

**Review by rustam-tolipov — APPROVED** [review]
## Hi @KayLemba 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by KayLemba** [comment]
Thanks for the review

**Pull request merged**